### PR TITLE
[VEN-2484]: configure EURA Price 

### PR DESCRIPTION
**Pull request by web3rover**
## Description

Resolves VEN-2484

### CODE REVIEW

File: vips/vip-273/bscmainnet.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    abstainDescription: "Indifferent to execution",
+  };
+
+  return makeProposal(
+    [
+      {
+        target: BINANCE_ORACLE,
+        signature: "setMaxStalePeriod(string,uint256)",
+        params: ["EURA", 1500],
+      },
```
Comment (chechu) [inline vips/vip-273/bscmainnet.ts:26]: I think we have to remove the current overriding EURA -> AGEUR

File: simulations/vip-273/bscmainnet.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+      const symbol = await vagEURContract.symbol();
+      expect(symbol).equals("vEURA_Stablecoins");
+    });
+
+    it("Check Prices", async () => {
+      await setMaxStalePeriodInBinanceOracle(BINANCE_ORACLE, "EURA");
+
+      const priceHAY = await resilientOracle.getUnderlyingPrice(vagEUR);
+      expect(priceHAY).equals(parseUnits("1.08551962", 18));
```
Comment (chechu) [inline simulations/vip-273/bscmainnet.ts:73]:
```suggestion
      const priceagEUR = await resilientOracle.getUnderlyingPrice(vagEUR);
      expect(priceagEUR).equals(parseUnits("1.08551962", 18));
```

File: simulations/vip-273/bsctestnet.ts [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+      const symbol = await vagEURContract.symbol();
+      expect(symbol).equals("vEURA_Stablecoins");
+    });
+
+    it("Check Prices", async () => {
+      await setMaxStalePeriodInBinanceOracle(BINANCE_ORACLE, "EURA");
+
+      const priceHAY = await resilientOracle.getUnderlyingPrice(vagEUR);
+      expect(priceHAY).equals(parseUnits("1.06", 18));
```
Comment (chechu) [inline simulations/vip-273/bsctestnet.ts:68]:
```suggestion
      const priceagEUR = await resilientOracle.getUnderlyingPrice(vagEUR);
      expect(priceagEUR).equals(parseUnits("1.06", 18));
```

File: simulations/vip-273/bscmainnet.ts [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+      await setMaxStalePeriodInBinanceOracle(BINANCE_ORACLE, "EURA");
+
+      const priceagEUR = await resilientOracle.getUnderlyingPrice(vagEUR);
+      expect(priceagEUR).equals(parseUnits("1.08551962", 18));
+    });
+
+    it("Verify symbol override", async () => {
+      const symbolOverride = await binanceOracle.symbols("EURA");
+      expect(symbolOverride).equals("");
+    });
```
Comment (kkirka) [inline simulations/vip-273/bscmainnet.ts:79]: Ideally also check that the implementation == initial impementation

**Review by kkirka — APPROVED** [review]

**Review by chechu — APPROVED** [review]

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/278?chainId=56

**Pull request merged**